Cargo.toml

```diff
@@ -50,15 +45,15 @@ rand_xorshift = "0.3"
 rand_chacha = "0.3"
 rand = "0.8"
 rand_core = { version = "0.6", default-features = false }
-sha1 = { version = "0.10.1", default-features = false }
-sha2 = { version = "0.10.2", default-features = false }
+sha1 = { version = "0.10.5", default-features = false, features = ["oid"] }
+sha2 = { version = "0.10.6", default-features = false, features = ["oid"] }
 sha3 = { version = "0.10.1", default-features = false }
```

Comment:
Maybe it's worth to add tests for SHA-3 OIDs as well?

Reply:
I've added the sha-3 test